vms/volunteer/utils.py

```diff
@@ -0,0 +1,19 @@
+from functools import wraps
+from django.shortcuts import render
+from django.http import Http404
+from volunteer.services import get_volunteer_by_id
+
+def vol_id_check(func):
+    @wraps(func)
+    def wrapped_view(request, volunteer_id):
+        vol = getattr(request.user, 'volunteer', hasattr(request.user, 'administrator'))
```

Comment:
@necessary129 Yes, but what I meant was why not simply check if the user has volunteer attribute. If they don't , then simply display the no rights page. And in case he/she is a volunteer, the ids are checked. Did you intend to allow the administrator  to view these pages? Because currently, they are able to and you haven't handled the case where vol=True (i.e. he/she is an administrator)
Don't make changes yet for this, I just want to know your views on it. :)